backend/graph/schema.graphqls

```diff
@@ -247,6 +247,14 @@ type FAQCategory {
   questions: [FAQ!]!
 }
 
+type User {
+  personId: ID
```

Comment:
Do we want this to be called personId? One could argue that this is a bad naming originally in Members...? 

I'd either go with "id" or "userId".

Reply:
Of course if this is just a "forward" of what we get from auth0, then we don't have much of a choice

Reply:
Yeah, I'll move this to ID, makes more sense, even if members i "wrong".